src/main/java/org/springframework/data/couchbase/repository/support/SimpleReactiveCouchbaseRepository.java

```diff
@@ -238,7 +238,8 @@ protected Flux mapFlux(Observable observable) {
                 .queryView(query)
                 .flatMap(AsyncViewResult::rows)
                 .map(asyncViewRow ->
-                        Long.valueOf(asyncViewRow.value().toString())).toSingle());
+                        Long.valueOf(asyncViewRow.value().toString()))
+                .switchIfEmpty(Observable.just(Long.valueOf(0L))).toSingle());
```

Comment:
NIT: Instead of `Observable.just(Long.valueOf(0L))` you can take advantage of auto-boxing and write `Observable.just(0L)`.

Reply:
ohhh nice - I see that now.  Will update now!